pubspec.yaml

```diff
@@ -0,0 +1,10 @@
+name: dio_workspace
```

Comment:
- https://github.com/invertase/melos/blob/main/pubspec.yaml
- https://github.com/firebase/flutterfire/blob/master/pubspec.yaml

Reply:
Yea they all do this because they use `melos`. But I think it is a good idea.

Reply:
But this involves us changing Github workflows to run `pub get` in the root repository before anything else.